docs/CONTRIBUTING.md

```diff
@@ -44,12 +44,23 @@ Use the following steps to ensure everything goes flawlessly:
 - Install the new version of the dependency. For example: `npm install --save
   <package>@<version>`. This will update the `npm-shrinkwrap.json` file.
 
-- Run `npm run shrinkwrap`. This is a small script that ensures that operating
-  system specific dependencies that could get included in the previous step are
-  removed from `npm-shrinkwrap.json`.
+- Run `npm run clean-shrinkwrap`. This is a small script that ensures that
+  operating system specific dependencies that could get included in the
+  previous step are removed from `npm-shrinkwrap.json`.
 
 - Commit *both* `package.json` and `npm-shrinkwrap.json`.
 
+Some npm versions seem to contain an issue were all development dependencies
```

Comment:
Should these instructions be *above* the "Commit *both* `package.json` and `npm-shrinkwrap.json`" line?